### PR TITLE
Text improvement and correctional changes

### PR DESCRIPTION
**Pull request by aliahcodes**

### CODE REVIEW

File: mevislab.github.io/content/tutorials/summary/summary1.md [hunk 1]
```diff
@@ -36,44 +36,41 @@ Add a `LocalImage` and a `View2D` module to your workspace. You are now able to
...
-
+Open the `SoView2DOverlay` module, change Blend Mode to *Blend* and select any color and *Alpha Factor* for your overlay. The applied changes are immediately visible. 
 ![Overlay color and transparency](/images/tutorials/summary/Example1_3.png "Overlay color and transparency")
 
-The segmented results from the `RegionGrowing` module might contain some holes because of differences in the intensity value of neighboring pixels. You can close those holes by using a `CloseGap` module. Connect it between the `RegionGrowing` and the `SoView2DOverlay` and configure Filter Mode as *Binary Dilatation*, Border Handling as *Pad Dst Fill* and set KernelZ to *3*.
+The segmented results from the `RegionGrowing` module might contain some holes because of differences in the intensity value of neighboring pixels. You can close these holes by adding a `CloseGap` module. Connect it to the `RegionGrowing` and the `SoView2DOverlay` module and configure Filter Mode as *Binary Dilatation*, Border Handling as *Pad Dst Fill* and set KernelZ to *3*.
```
Comment (mrbean-bremen) [inline mevislab.github.io/content/tutorials/summary/summary1.md:52]: holes -> gaps

File: mevislab.github.io/content/tutorials/summary/summary1.md [hunk 2]
```diff
@@ -17,17 +17,17 @@ menu:
...
-In this example, we will develop a network which fulfills the requirements from the [overview page](/tutorials/summary#DevelopNetwork). The network will be developed by re-using existing modules and defining basic field values.
+In this example, we will develop a network which fulfills the requirements mentioned on the [overview page](/tutorials/summary#DevelopNetwork). The network will be developed by re-using existing modules and defining basic field values.
 
 ## Steps to do
 ### 2D viewer
-The 2D viewer shall visualize the loaded images. In addition to that, it shall be possible to click into the image for starting a RegionGrowing algorithm segmenting parts of the loaded image based on a threshold.
+The 2D viewer shall visualize the loaded images. In addition to that, it shall be possible to click into the image to trigger a RegionGrowing algorithm to segment parts of the loaded image based on a threshold.
 
-The following requirements from [Overview](/tutorials/summary#DevelopNetwork) will be implemented:
+The following requirements from [overview](/tutorials/summary#DevelopNetwork) will be implemented:
```
Comment (mrbean-bremen) [inline mevislab.github.io/content/tutorials/summary/summary1.md:26]: from overview -> from the overview

File: mevislab.github.io/content/tutorials/summary/summary1.md [hunk 3]
```diff
@@ -49,7 +49,7 @@ The overlay is shown in white.
 Open the `SoView2DOverlay` module, change Blend Mode to *Blend* and select any color and *Alpha Factor* for your overlay. The applied changes are immediately visible. 
 ![Overlay color and transparency](/images/tutorials/summary/Example1_3.png "Overlay color and transparency")
 
-The segmented results from the `RegionGrowing` module might contain some holes because of differences in the intensity value of neighboring pixels. You can close these holes by adding a `CloseGap` module. Connect it to the `RegionGrowing` and the `SoView2DOverlay` module and configure Filter Mode as *Binary Dilatation*, Border Handling as *Pad Dst Fill* and set KernelZ to *3*.
+The segmented results from the `RegionGrowing` module might contain some holes because of differences in the intensity value of neighboring pixels. You can close these gaps by adding a `CloseGap` module. Connect it to the `RegionGrowing` and the `SoView2DOverlay` module and configure Filter Mode as *Binary Dilatation*, Border Handling as *Pad Dst Fill* and set KernelZ to *3*.
```
Comment (mrbean-bremen) [inline mevislab.github.io/content/tutorials/summary/summary1.md:52]: still holes left :)

Comment (aliahcodes) [reply mevislab.github.io/content/tutorials/summary/summary1.md:52]: Overlooked that one, sorry :)

**Pull request merged**